tests/unit/services/gmail/test_unsubscribe.py

```
@@ -0,0 +1,253 @@
+        result = unsubscribe_single("example.com", "https://example.com/unsub")
+
+        assert result["success"] is True
+
+    @patch("app.services.gmail.unsubscribe.validate_unsafe_url")
+    @patch("app.services.gmail.unsubscribe.urllib.request.urlopen")
+    def test_post_fails_get_succeeds(self, mock_urlopen, mock_validate):
+        """POST failure should fallback to GET."""
+        mock_validate.return_value = "https://example.com/unsub"
+
+        # POST fails, GET succeeds
+        post_response = MagicMock()
```

Ruff (0.14.10)

Local variable `post_response` is assigned to but never used

Remove assignment to unused variable `post_response`

(F841)